src/crypto.js

```diff
@@ -109,6 +109,64 @@ class Crypto {
     }
   }
 
+  async transactionInfo() {
+    const cryptoNames = {
+      1: 'Bitcoin',
+      2: 'Dash',
+      3: 'Dogecoin',
+      4: 'Litecoin',
+    };
+    const abbreviation = {
+      1: 'btc',
+      2: 'dash',
+      3: 'doge',
+      4: 'ltc',
+    };
+
+    console.log('\nList of cryptos:');
+    for (const key in cryptoNames) {
+      console.log(`${key}. ${cryptoNames[key]}`);
+    }
+
+    const chosenCrypto = await promised.question('\nEnter the number' +
+      ' of crypto from the list above you\'d to like to input hash of: \n');
+    const hash = await promised.question('\nEnter the hash of ' +
+      'transaction you\'d like to get info about: \n');
+
+    if (hash.length !== 64) {
+      console.log(`${red}Wrong hash${green}`);
+      return;
+    }
+
+    const info = await safeGet(`https://api.blockcypher.com/v1/${abbreviation[chosenCrypto]}/main/txs/${hash}`);
+    const keys = [
+      'total',
+      'fees',
+      'size',
+      'preference',
+      'received',
+    ];
+    const outputKeys = [
+      '\nSatoshis sent',
+      'Fee in satoshis',
+      'Transaction size in bytes',
+      'Transaction preference',
+      'Received at',
+    ];
+    if (Object.prototype.hasOwnProperty.call(info, 'error')) {
+      console.log(`${red}Wrong hash${green}`);
+      return;
+    } else if (Object.prototype.hasOwnProperty.call(info, 'confirmed')) {
+      keys.push('confirmed');
+      outputKeys.push('Confirmed at');
+    } else console.log('\nTransaction isn\'t confirmed yet :C');
```

Comment:
```suggestion
    } else {
      console.log('\nTransaction isn\'t confirmed yet :C');
    }
```
Just for code style consistency